src/index.js

```diff
@@ -11,78 +11,75 @@ function Square(props){
 }
 
 class Board extends React.Component {
-  renderSquare(i) {
-    return (
-      <Square
-        value={this.props.squares[i]}
-        onClick={() => this.props.onClick(i)}
-      />
-    );
-  }
 
   render() {
+    const board = [[0,1,2],[3,4,5],[6,7,8]];
+
     return (
       <div>
-        <div className="board-row">
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
-        </div>
-        <div className="board-row">
-          {this.renderSquare(3)}
-          {this.renderSquare(4)}
-          {this.renderSquare(5)}
-        </div>
-        <div className="board-row">
-          {this.renderSquare(6)}
-          {this.renderSquare(7)}
-          {this.renderSquare(8)}
+      {board.map(row => (
+        <div className="board-row" key={row}>
+          {row.map(square => (
+            <Square
+              key={square}
+              value={this.props.squares[square]}
+              onClick={() => this.props.onClick(square)}
+            />
+          ))}
         </div>
+      ))}
       </div>
     );
   }
 }
 
 class Game extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
```

Comment:
Checking the docs, it states that if you are going to override con constructor with `super(props)` why in this case is without arguments?